recipes/simbody/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conan import ConanFile, tools
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.scm import Version
+import os
+import textwrap
+
+
+class SimbodyConan(ConanFile):
+    name = "simbody"
+    description = "High-performance, open-source toolkit for science- and engineering-quality simulation"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/simbody/simbody"
+    topics = ("high-performance", "science", "simulation")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "CMakeLists.txt"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+         "build_lapack": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "build_lapack": True
+    }
+
+    def validate(self):
+        if self.options.build_lapack:
+            raise ConanInvalidConfiguration(
+                "build simbody from sources locally with default_options \"openblas:build_lapack\": True"
+            )
+
```

Comment:
```suggestion
        if not self.dependencies["openblas"].options.build_lapack:
            raise ConanInvalidConfiguration(f"{self.name} requires the openblas:build_lapack": True")
```
OpenBlas need that option, not this recipe.